lib/rules/jsx-curly-spacing.js

```diff
@@ -32,7 +32,52 @@ module.exports = {
     fixable: 'code',
 
     schema: [{
-      enum: SPACING_VALUES
+      definitions: {
```

Comment:
I've just read in the ESLint docs that it's using JSON schema - then went over to the JSON schema reference page ([this one is very good](https://spacetelescope.github.io/understanding-json-schema/)) and saw that this technique there.

After a little investigation, it should be supported since the schema validation itself was added: https://github.com/eslint/eslint/commit/fd8fbe721781ce26957ad0f0b3ffbb33201ac387. According to the changelog, the version was 0.22.0, released on 2015-05-30.

The version of is-my-json-valid used then is 2.10.0, which already handled refs: https://github.com/mafintosh/is-my-json-valid/blob/21aa9b8916a2efb3172d17d2d3d1b517668834c7/test/json-schema-draft4/ref.json

Btw. "definitions" is just a convention; the ref can reference any path in the schema.